Components/AccordionDefault/_style.scss

```diff
@@ -1,5 +1,13 @@
 [is='flynt-accordion-default'] {
-  margin: $component-spacing-desktop 0;
+  margin: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    margin: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    margin: $component-spacing-desktop 0;
+  }
```

Comment:
Please refer to #69.

Reply:
Ignored it as per the following comment https://github.com/flyntwp/flynt/pull/69#discussion_r302961405